Makefile

```diff
@@ -30,12 +30,15 @@ mod:
 generate: mod
 	@echo "Running cli on version: $(VERSION)"
 	@GO111MODULE=on GLFLAGs=-mod-vendor go run cmd/generator/generator.go $(ARGS)
+	@GO111MODULE=on GLFLAGs=-mod-vendor go run cmd/sbomgen/sbomgen.go $(ARGS) > examples/sbom.spdx
```

Comment:
I've added a command alongwith the existing Makefile command to run the refactor to generate a SBOM in the existing examples directory.
This generated SBOM is validated later by the GitHub action.